lib/tasks/server/livereload-server.js

```diff
@@ -5,6 +5,10 @@ var path        = require('path');
 var Task        = require('../../models/task');
 var SilentError = require('../../errors/silent');
 
+// default reload css extensions
+var styleExtensions = ['css', 'scss', 'sass', 'less', 'styl'];
```

Comment:
We can get the known extensions for styles from the registry (something similar is done in the addon model with template extensions that you can probably tweak).